applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -179,17 +179,27 @@ Check and configure the following configuration options for the sample:
 
    This option enables additional AT commands for using the SMS service.
 
-.. option:: CONFIG_SLM_GPS - GPS support in SLM
+.. option:: CONFIG_SLM_GNSS - GNSS support in SLM
 
-   This option enables additional AT commands for using GPS service.
+   This option enables additional AT commands for using GNSS service.
 
-.. option:: CONFIG_SLM_SUPL_SERVER - SUPL server
+.. option:: CONFIG_SLM_AGPS - nRF Cloud A-GPS support in SLM
 
-   This option specifies the SUPL server to use for retrieving SUPL A-GPS data.
+   This option enables additional AT commands for using nRF Cloud A-GPS service.
 
-.. option:: CONFIG_SLM_SUPL_PORT - SUPL server port
+   This option is not selected by default.
+
+.. option:: CONFIG_SLM_PGPS - nRF Cloud P-GPS support in SLM
+
+   This option enables additional AT commands for using nRF Cloud P-GPS service.
+
+   This option is not selected by default.
+
+.. option:: CONFIG_SLM_CELL_POS - nRF Cloud cellular positioning support in SLM
 
-   This option specifies the port to use for the specified SUPL server.
+   This option enables additional AT commands for using nRF Cloud cellular positioning service.
```

Comment:
```suggestion
   This option enables additional AT commands for using the nRF Cloud Cellular Positioning service.
```
I think the service name must be capitalized. And again the same with the next line.